cursorless-talon/src/spoken_forms.py

```diff
@@ -0,0 +1,97 @@
+import json
+from pathlib import Path
+
+from talon import app
+
+from .csv_overrides import SPOKEN_FORM_HEADER, init_csv_and_watch_changes
+from .marks.mark import init_marks
+from .paired_delimiter import paired_delimiter_spoken_form_defaults
+
+JSON_FILE = Path(__file__).parent / "spoken_forms.json"
+
+
+def on_ready() -> None:
+    with open(JSON_FILE) as file:
```

Comment:
I wonder if we want to use talon resource tracker or something. During development of our cursorless-talon files, we won't trigger a reload of this file if it changes, so not ideal DX.  I remember there being some gotchas on resource tracker that were fixed in knausj in last 8 months tho so may be worth an issue search if we decide it's worth the effort here

Reply:
Good call. I will look into that. It requires 0.4 I believe, but I think its reasonable to expect people to update talon and cursorless.

Reply:
Might be worth a look at the community 0.4 update PR to see if they're trying to detect old talon version / holding off on merge. I guess we could just detect old talon and fall back; prob not too hard

Reply:
Pretty sure the community pr doesn't do that but we could of course.

Reply:
File watcher added. I decided to go with the old one I'm let community the merge first. Once community require the latest version of Talon I think Cursorless is free to do the same.